extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfigBuilder.java

```diff
@@ -1239,4 +1253,101 @@ public Token getToken() {
     public Logout getLogout() {
         return logout;
     }
+
+    /**
+     * Merge the current tenant (represented by this builder) and well-known OpenId Connect provider configurations.
+     * Initialized properties take priority over uninitialized properties.
+     *
+     * Initialized properties in the current tenant configuration take priority
+     * over the same initialized properties in the well-known OpenId Connect provider configuration.
+     *
+     * @param tenant well-known OpenId Connect provider configuration
+     */
+    public OidcTenantConfigBuilder mergeTenant(io.quarkus.oidc.OidcTenantConfig tenant) {
```

Comment:
@michalvavrik Michal, I'm still not sure this one belongs here. It is a `utility` method, and not part of the `OidcTenantConfigBuilder` API.  The reason it is not part of this API is that users who build `OidcTenantConfig` with this builder won't be able to use this method for building this config instance, and therefore it does not belong in this builder API.

Reply:
ok, it will bring extra processing to move it, but I'll move it, not a problem (I expected it :-) )

Reply:
Done.